csrc/scheduler/pointwise.cpp

```diff
@@ -197,10 +200,22 @@ std::shared_ptr<PointwiseParams> getPointwiseHeuristics(
         (int64_t)dataTypeSize(inp->getDataType().value(), index_type));
   }
 
+  std::unordered_map<int, int> rfactor_reorder_map;
```

Comment:
`data_cache` is available in this function. Could you cache the computation of `rfactor_reorder_map`?